examples/InvoiceExample.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Xendit\Xendit;
+
+require 'vendor/autoload.php';
+
+Xendit::setApiKey(
+    <<<'TAG'
+xnd_development_prHUBDfVuOQTxyWTQSNkpjn9OwX9ZSUjdqgF9GenZ6hwhUQkc3NZ9WVexdH
+TAG
+);
+
+$params = ['external_id' => 'demo_147580196270',
+    'payer_email' => 'sample_email@xendit.co',
+    'description' => 'Trip to Bali',
+    'amount' => 32000
+];
+
+$createInvoice = \Xendit\Invoice::create($params);
+var_dump($createInvoice);
+
+$id = $createInvoice['id'];
+
+$getInvoice = \Xendit\Invoice::retrieve($id);
+var_dump($getInvoice);
+
+$expireInvoice = \Xendit\Invoice::expireInvoice($id);
+var_dump($expireInvoice);
+
+//$getAllInvoice = \Xendit\Invoice::retrieveAll();
```

Comment:
If you don't use it anymore, I think we can remove these comments :) to make our code cleaner